content/en/docs/collector/internal-telemetry.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Internal Telemetry
+weight: 25
+cSpell:ignore: journalctl kube otecol pprof tracez zpages
+---
+
+The Collector offers multiple ways to measure and monitor its own health. In
+this section, you'll learn how to enable internal observability.
+
+## Enabling observability internal to the Collector
+
+By default, the Collector exposes service telemetry in two ways:
+
+- Internal [metrics](#configuring-metrics) are exposed using a Prometheus
+  interface which defaults to port `8888`.
+- [Logs](#configuring-logs) are emitted to `stdout`.
+
+[Traces](#configuring-traces) are not exposed by default but two feature gates
+offer experimental support for a configuration based on the OpenTelemetry
+Configuration schema.
+
+### Configuring metrics
+
+Prometheus metrics are exposed locally on port `8888` and path `/metrics`. For
+containerized environments, you may want to expose this port on a public
+interface instead of only locally.
+
+Set the address in the config `service::telemetry::metrics`:
+
+```yaml
+service:
+  telemetry:
+    metrics:
+      address: ':8888'
+```
+
+To visualize these metrics, you can use the
+[Grafana dashboard](https://grafana.com/grafana/dashboards/15983-opentelemetry-collector/),
+for example.
+
+You can enhance the metrics telemetry level using the `level` field. The
+following is a list of all possible values and their explanations.
+
+- `none` indicates that no telemetry data should be collected.
+- `basic` is the recommended value and covers the basics of the service
+  telemetry.
+- `normal` adds other indicators on top of basic.
+- `detailed` adds dimensions and views to the previous levels.
+
+For example:
+
+```yaml
+service:
+  telemetry:
+    metrics:
+      level: detailed
+      address: ':8888'
+```
+
+The Collector can also be configured to scrape its own metrics and send them
+through configured pipelines. For example:
```

Comment:
Should we add a note here about how doing this is a bit dangerous? Monitoring yourself is usually frowned upon since if something goes wrong, you can't trust yourself to reliably transport the observability data

Reply:
I think our distro does that by default: https://github.com/signalfx/splunk-otel-collector/blob/main/cmd/otelcol/config/collector/agent_config.yaml#L69-L85 

+CC @atoulme

Reply:
Agreed with @mx-psi, this needs to be called out (and ideally we point out best practices, https://tractatus.one/otelcol-metrics-pipeline-3d161983b5b9 comes to mind?)

Reply:
Here's what I've come up with for a 'Caution' callout. Please let me know if I should make any changes or additions.

> Self-monitoring is a risky practice. If an issue arises, the source of the problem and the reliability of telemetry data become questionable.

Reply:
A simple warning like that is a good starting point

Reply:
Added note in [625af6f](https://github.com/open-telemetry/opentelemetry.io/pull/4246/commits/625af6f18d342df25c5418c89e05e745cb565c68). Resolving conversation.